charts/fiberplane-daemon/values.yaml

```diff
@@ -0,0 +1,43 @@
+nameOverride: ""
+fullnameOverride: ""
+
+image:
+  repository: fiberplane/fpd
+  pullPolicy: IfNotPresent
+  tag: ""
+
+fiberplane:
+  token: ""
+  data-sources: []
+
+imagePullSecrets: []
+
+serviceAccount:
+  create: false
+  annotations: {}
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+
+securityContext: {}
+
+service:
```

Comment:
This is not present anymore, right? Same with the ingress and possibly other values in here.

Reply:
I left that as it's something that can be defined in the `deployment.yaml` in case people have stricter configurations here?

Reply:
If it is not being used in our helm template, then we should just get rid of it in the values files, right?

Reply:
It is here: https://github.com/fiberplane/helm-charts/blob/1ccb84492f61866913ed9370e2af51d1940a6bc7/charts/fiberplane-daemon/templates/deployment.yaml#L28